rust/numaflow-core/src/monovertex.rs

```diff
@@ -18,7 +18,7 @@ use crate::{metrics, shared};
 /// - Invokes the SourceTransformer concurrently
 /// - Calls the Sinker to write the batch to the Sink
 /// - Send Acknowledgement back to the Source
-mod forwarder;
+pub(crate) mod forwarder;
```

Comment:
Any specific reason for making it pub?

Reply:
we need it for using in end to end test in sqs